<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="account_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Аккаунт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>311</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="species_input"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="class_input"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Раса:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Класс:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Уровень</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Опыт</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Мировоззрение</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Предыстория:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="background_input"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="worldview_input"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Имя персонажа</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_input"/>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="exp_input">
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="level_input">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url('data/textures/background.jpg');
background-repeat: no-repeat; 
background-position: center;</string>
    </property>
    <widget class="QPushButton" name="throw_dice_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
     </property>
     <property name="text">
      <string>Кинуть дайс</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </widget>
   <zorder>widget</zorder>
   <zorder>account_btn</zorder>
   <zorder>back_btn</zorder>
   <zorder>gridLayoutWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
